Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import "./Button.css";
 
```

Comment:
great thinking, to getting the numbers in a dry way, im thinking you are going to want to map that into this component. one suggestion i may add at this point is to carefully look at the read me, you may want to set up the array inside app because you are going to pass the number(text) as props to the components.